prime-router/src/test/kotlin/fhirengine/azure/FHIRConverterIntegrationTests.kt

```diff
@@ -377,6 +378,7 @@ class FHIRConverterIntegrationTests {
             val actionLogs = DSL.using(txn).select(Tables.ACTION_LOG.asterisk()).from(Tables.ACTION_LOG)
                 .where(Tables.ACTION_LOG.REPORT_ID.eq(receiveReport.id))
                 .and(Tables.ACTION_LOG.TYPE.`in`(ActionLogType.error, ActionLogType.warning))
+                .orderBy(Tables.ACTION_LOG.ACTION_LOG_ID.asc())
```

Comment:
Since the query plan is now using the index to fetch the rows, the ordering changed.  I've fixed it permanently by adding a specific `orderBy` clause